.ci/wercker.sh

```diff
@@ -52,10 +52,16 @@ no-error-pgjdbc)
   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   echo CS_version: ${CS_POM_VERSION}
+  ls -l /root/.m2
+  ls -l /root/.m2/repository
+  ls -l /root/.m2/repository/com/puppycrawl/tools/checkstyle
+  ls -l /root/.m2/repository/com/puppycrawl/tools/checkstyle/8.31-SNAPSHOT
```

Comment:
> CS_version: 8.31-SNAPSHOT
total 12
-rw-r--r--. 1 root root  129 Mar  9 15:24 copy_reference_file.log
drwxr-xr-x. 2 root root 4096 Mar  9 15:24 repository
-rw-r--r--. 1 root root  327 Mar  9 15:24 settings-docker.xml
total 0
lrwxrwxrwx. 1 root root 15 Mar  9 15:24 cache -> /pipeline/cache
ls: cannot access '/root/.m2/repository/com/puppycrawl/tools/checkstyle': No such file or directory

As we discussed earlier, checkstyle isn't stored into the `.m2` folder.